File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppJava/app/src/main/java/pl/udu/uwr/pum/verynobleappjava/data/laureateresponse/nobelprize/affiliation/Affiliation.java
package pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.affiliation;

import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.Country;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.CountryNow;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.LocationString;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.Name;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.NameNow;

public class Affiliation {
    public Name name;
    public NameNow nameNow;
    public City city;
    public Country country;
    public CityNow cityNow;
    public CountryNow countryNow;
    public LocationString locationString;
}

File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppKotlin/app/src/main/java/pl/udu/uwr/pum/verynobleappkotlin/data/laureateresponse/nobelprize/affiliation/Affiliation.kt
package pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.affiliation

import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.*

data class Affiliation(
    val city: City,
    val cityNow: CityNow,
    val country: Country,
    val countryNow: CountryNow,
    val locationString: LocationString,
    val name: Name,
    val nameNow: NameNow
)